lib/clients/BaseClient.py

```diff
@@ -77,7 +77,6 @@ def _get_credentials_from_credhub(self, *args):
         return self.__retry(self.__get_credentials_from_credhub, params)
     
     def __get_credentials_from_credhub(self, configuration, access_token):
-        access_token = self.__getAccessToken(configuration)
```

Comment:
One question, if retrying access toke, fails [here](https://github.com/cloudfoundry-incubator/service-fabrik-backup-restore/pull/39/files#diff-973b6bc10f6b36c84d0dbeb7f11dc11aL75), looks like it returns `None`, [here](https://github.com/cloudfoundry-incubator/service-fabrik-backup-restore/blob/dcaac2efd9387ef2720670ec5dc539e99f7ffb6d/lib/clients/BaseClient.py#L149). Should we add a check then before calling `__get_credentials_from_credhub` ?

Reply:
@ketakig12 that check seemed unnecessary as the method `requests.get` on LN 84 will anyways throw an runtime error of invalid access token. Since retry is in place, the occurrence of this going to be rare and in those rare scenarios, its OK for this runtime error from LN84.

Reply:
Okay.